packages/siri-sx-generator/util/siri.ts

```diff
@@ -60,12 +61,14 @@ const getPeriod = (period: Validity): Period => ({
         : {}),
 });
 
-export const getPtSituationElementFromSiteDisruption = (
+export const getPtSituationElementFromSiteDisruption = async (
     disruption: Disruption & { organisation: { id: string; name: string } },
-) => {
+): Promise<PtSituationElement> => {
+    const adminAreas = await fetchAdminAreas();
```

Comment:
I think in this case it would be better to make the `fetchAdminAreas` API call the next level up and pass `adminAreas` through to the getPtSituationsElementFromSiteDisruption function. The reason for this, is so we only make one API call instead of one API per disruption (could slow the generator down in prod)

Reply:
done